scripts/preprocessing/04b-run_ssp.py

```diff
@@ -21,6 +23,30 @@
 
 
 @failsafe_run(on_error=on_error)
+
+def _get_global_reject_ssp(raw):
+    if 'eog' in raw:
+        eog_epochs = mne.preprocessing.create_eog_epochs(raw)
+    else:
+        eog_epochs = []
+    if len(eog_epochs) >= 5:
```

Comment:
this is btw. a somewhat arbitrary choice, I just decided that with fewer than 5 events you should not try to estimate an SSP projector.

Reply:
perhaps we can just add a comment for now.